sentinel-cluster/sentinel-cluster-client-redis/src/main/java/com/alibaba/csp/sentinel/cluster/redis/config/RedisFlowRuleManager.java

```diff
@@ -0,0 +1,137 @@
+package com.alibaba.csp.sentinel.cluster.redis.config;
+
+import com.alibaba.csp.sentinel.cluster.redis.RedisClient;
+import com.alibaba.csp.sentinel.cluster.redis.RedisClientFactory;
+import com.alibaba.csp.sentinel.log.RecordLog;
+import com.alibaba.csp.sentinel.property.DynamicSentinelProperty;
+import com.alibaba.csp.sentinel.property.PropertyListener;
+import com.alibaba.csp.sentinel.property.SentinelProperty;
+import com.alibaba.csp.sentinel.slots.block.flow.ClusterFlowConfig;
+import com.alibaba.csp.sentinel.slots.block.flow.FlowRule;
+import com.alibaba.csp.sentinel.slots.block.flow.FlowRuleUtil;
+import com.alibaba.csp.sentinel.util.AssertUtil;
+
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class RedisFlowRuleManager {
+    public static volatile boolean publishRuleToRedis = true;
+
+    private static Map<String, List<FlowRule>> flowRules = new HashMap<String, List<FlowRule>>();
+    private static Map<Long, FlowRule> flowIdToRule = new HashMap();
```

Comment:
A diamond should be added here to get rid of warning.

Reply:
Err......Actually the warning is in `new HashMap()` which should better be `new HashMap<>()`